modules/apps/collaboration/item-selector/item-selector-taglib/src/main/resources/META-INF/liferay-item-selector.tld

```diff
@@ -11,15 +11,18 @@
 	<short-name>liferay-item-selector</short-name>
 	<uri>http://liferay.com/tld/item-selector</uri>
 	<tag>
+		<description>A User interface to select groups<![CDATA[<br /><br />Example:<br /><img src="../images/item-selector/group-selector.png"/>]]></description>
```

Comment:
By groups, do you mean scope groups, similar to an [application scope](https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/application-scope)?